fontforge/python.c

```diff
@@ -18811,7 +18971,7 @@ return( NULL );
     self->is_quadratic = PyInt_AsLong(PyTuple_GetItem(args,0));
     if ( PyErr_Occurred()!=NULL )
 return( NULL );
-    self->cntr_cnt = self->cntr_max = len-2;
+    self->cntr_cnt = self->cntr_max = len-1;
```

Comment:
Was this a bug or due to a change elsewhere?

Reply:
Just a bug I ran into when testing.

Reply:
I remember this now. There's a zip file above with some python code to store and retrieve custom convex nibs for the UI, using the python interface I added for that purpose. It stores its data by pickling a dictionary that contains `fontforge.layer`s. That's how I found and fixed this bug -- the pickling didn't work without the change.

Reply:
This went in as a separate fix to aid in testing.